hibike/devices/YogiBear/YogiBear.cpp

```diff
@@ -63,43 +64,19 @@ void loop() {
 
 uint32_t device_write(uint8_t param, uint8_t* data, size_t len) {
   switch (param) {
-    case ENABLE:
-      if (data[0] == 0) {
-        motorDisable();
-      } else {
-        motorEnable();
-      }
-      return sizeof(bool);
-      break;
-
-    case COMMAND_STATE: 
-      driveMode = data[0];
-
-      switch (driveMode) {
-        case MANUALDRIVE:
-          disablePID();
-          break;
-        case PID_VEL:
-          enableVel();
-          break;
-        case PID_POS:
-          enablePos();
-          break;
-        default:
-          driveMode = MANUALDRIVE;
-          disablePID();
-          break;
-      }
 
-      return sizeof(uint8_t);
-      break;
-
-    case DUTY_CYCLE: 
+    case DUTY_CYCLE:
+      motorEnable();
+      disablePID(); 
+      driveMode = MANUALDRIVE;
       pwmInput = ((float *)data)[0];
       return sizeof(float);
       break;
 
     case PID_POS_SETPOINT: 
+      motorEnable();
+    	driveMode = PID_POS;
+      enablePos();
```

Comment:
Is this in reference to the enablePos or motorEnable? in both cases this is because we removed the ENABLE and COMMAND_STATE params. Originally we gave students direct control over those, but decided to automatically switch those two states based on which type of input they were using to drive the robot.